### DISCUSSION TITLE
Launch to favorites

### DISCUSSION THREAD

**Discussion by AndreZoro** [eduardofilo/RG350_adam_image#147] [SimpleMenu topics]
Hi,
I have setup a handheld for my five year old son. He does not care if a game used to be on Sega or a Gameboy. So for him having the games divided by device type and then console does not make much sense. It would be cool, if it would be possible to boot up to something like an all games list. Maybe an easy way to implement it would be by booting into the favorites?

**Comment by eduardofilo** [comment]
The version in which the SimpleMenu developer is working integrates the Favorites list as if it were one more system at the beginning of each group, so I suppose that if you stick to that entry, it will be the one that opens automatically on startup. But we will have to wait for it to be officially released.
